src/core/client/admin/test/community/community.spec.tsx

```diff
@@ -319,12 +319,11 @@ it("change user role to Site Moderator and add sites to moderate", async () => {
       .props.onClick();
   });
 
```

Comment:
It would be nice to have a test here for adding/removing `member` badges. Maybe we could even try to get the RTL changes in so that the test wouldn't be written just to be refactored?

It could also be nice to have a test for adding a comment in the stream and checking that the correct badge shows up if it's a member commenting.

Reply:
I'm making another PR for this that will do this very thing using the RTL.